src/composables/useRenderer/index.ts

```diff
@@ -122,7 +122,9 @@ export function useRenderer(
 ) {
 
   const webGLRendererConstructorParameters = computed<WebGLRendererParameters>(() => ({
-    alpha: toValue(options.alpha),
+    alpha: toValue(options.alpha) === undefined // an opinionated default of tres
```

Comment:
`toValue` returns `undefined` or a `boolean`, so it would be shorter and maybe more idiomatic to use a [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) instead of a ternary:

```ts
alpha = toValue(options.alpha) ?? true, // an opinionated default of tres
```